allennlp/service/config_explorer.py

```diff
@@ -74,132 +96,620 @@ def main(args):
     CORS(app)
 
     http_server = WSGIServer(('0.0.0.0', args.port), app)
-    print(f"Model loaded, serving demo on port {args.port}")
+    print(f"serving Config Explorer on port {args.port}")
     http_server.serve_forever()
 
-#
-# HTML and Templates for the default bare-bones app are below
-#
-_BASE_HTML_TEMPLATE = Template("""
+_HTML = """
+<!DOCTYPE html>
 <html>
-    <head>
-        <title>
-            $title
-        </title>
-        <style>
-            $css
-        </style>
-    </head>
-    <body>
-        $body
-    </body>
+  <head>
```

Comment:
I think you can avoid this duplication by adding the html file to `MANIFEST.in` in the root of the repo - this dictates non python stuff which is included/excluded by pip.

Reply:
I'm going to leave this for a future PR, I looked into it a bit and it seems like it's tricky to get right

Reply:
sure